bin/is-website-vulnerable.js

```diff
@@ -5,7 +5,9 @@
 const { Audit, RenderConsole } = require('../index')
 
 const url = process.argv[2]
-if (!url) {
+const regExp = RegExp(/^(http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?$/g)
```

Comment:
how about instead of validating if it's a URL or not, we can try to convert the url input provided into a URL object to parse it? See an example here: https://nodejs.org/api/url.html#url_url_protocol

Reply:
I added a new commit,

I had to use url.parse because with the URL object for the urls with no protocol returns an error.

What do you think?